lang/jamvm/Portfile

```diff
@@ -26,5 +28,5 @@ post-destroot {
 }
 
 livecheck.type	regex
-livecheck.url	http://sourceforge.net/projects/jamvm/files/
-livecheck.regex	${name}-(\[0-9.\]+)\\.tar
+livecheck.url	https://sourceforge.net/projects/jamvm/files/jamvm/JamVM%20${version}/
+livecheck.regex	${name}-(\[0-9.\]+)\\.tar.gz
```

Comment:
The upstream files indeed appears have been reorganized at some point into versioned subdirectories within the `files/jamvm` subdirectory, instead of directly all in one `files` directory.

But `${version}` should never appear in a livecheck, because that is the version the portfile currently uses (2.0.0 in this case); a future version won't appear in the subdirectory for the current version. We want it to look at the level where all available versions are listed, and then the livecheck mechanism will determine the greatest version available.

In this case, the Sourceforge RSS feed might be a better alternative that fetches less content and should find all versions regardless of how the files are organized:
```
livecheck.url       https://sourceforge.net/projects/jamvm/rss?path=/
livecheck.regex     ${name}-(\[0-9.\]+)\\.tar.gz
```